src/Symfony/Component/Webhook/Server/HeaderSignatureGenerator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Webhook\Server;
+
+use Symfony\Component\HttpClient\HttpOptions;
+use Symfony\Component\RemoteEvent\RemoteEvent;
+use Symfony\Component\Webhook\Exception\LogicException;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 6.3
+ */
+class HeaderSignatureGenerator implements GeneratorInterface
```

Comment:
All these "generators" could be final as they are easily decorable

Reply:
Done